recipes-core/initrdscripts/tegra-flash-init_1.0.bb

```diff
@@ -14,6 +14,8 @@ S = "${WORKDIR}"
 
 do_install() {
     install -m 0755 ${WORKDIR}/init-flash.sh ${D}/init
+    install -m 0755 ${WORKDIR}/init_extra.sh ${D}/init_extra
```

Comment:
The script to install is not included.
Also, there are inconsistency with underscore and hyphen, so that needs to be fixed as well.

Reply:
Updated PR with fixes. 
This has been verified with a clean build of core image with added recipe (custom-tegra-flash-init-extra):

```
DESCRIPTION = "Custom extra init for tegraflash initrd flashing"

LICENSE="CLOSED"

SRC_URI = "file://custom-extra-init.sh"

RDEPENDS:${PN} = " util-linux-findfs"

do_install () {
    install -m 0755 -d ${D}/init-extra.d
    install -m 0755 ${WORKDIR}/custom-extra-init.sh ${D}/init-extra.d/custom-extra-init
}

FILES:${PN} = "/"
```

And with distro config updated with:

```
TEGRA_INITRD_FLASH_INSTALL = "custom-tegra-flash-init-extra"
```